pkgs/tools/misc/tlp/default.nix

```diff
@@ -131,6 +132,10 @@ stdenv.mkDerivation rec {
       rm -rf $out/share/metainfo
     '';
 
+  passthru = {
+    features = { inherit enableRDW withSystemd; };
```

Comment:
not a review comment,

is this passthru.features documented anywhere? I only see a few uses of it in nixpkgs